test/integration/smart-loan.test.ts

```diff
@@ -424,5 +424,47 @@ describe('Smart loan', () => {
 
   });
 
+  describe('A loan with edge LTV cases', () => {
+    let loan: MockSmartLoan,
+      owner: SignerWithAddress;
+
+    before("deploy provider, exchange and pool", async () => {
+      [owner] = await getFixedGasSigners(10000000);
+    });
+
+    it("should deploy a smart loan", async () => {
+      loan = await (new MockSmartLoan__factory(owner).deploy());
+    });
+
+    it("should check debt equal to 0", async () => {
+      await loan.setValue(400);
+      await loan.setDebt(0);
+      expect(await loan.getLTV()).to.be.equal(0);
+      expect(await loan.isSolvent()).to.be.true;
+    });
+
+    it("should check debt greater than 0 and lesser than totalValue", async () => {
+      await loan.setValue(500);
+      await loan.setDebt(100);
+      expect(await loan.getLTV()).to.be.equal(250);
+      expect(await loan.isSolvent()).to.be.true;
+    });
+
+    it("should check debt equal to totalValue", async () => {
```

Comment:
Could we also 3 additional cases when:
1) Ltv is 4999
2) LTV is 5000 (calculated correctly)
3) LTV is 5001
?